packages/certifi/meta.yaml

```diff
@@ -0,0 +1,17 @@
+package:
+  name: certifi
+  version: 2022.6.15
+
+source:
+  url: https://files.pythonhosted.org/packages/cc/85/319a8a684e8ac6d87a1193090e06b6bbb302717496380e225ee10487c888/certifi-2022.6.15.tar.gz
```

Comment:
certifi has a pure python wheel. Could you replace the source from `.tar.gz` with `.whl`?

Reply:
Done.